src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -37,10 +37,10 @@ public Test()
 
                 SolutionTransforms.Add((solution, projectId) =>
                 {
-                    var parseOptions = (CSharpParseOptions)solution.GetRequiredProject(projectId).ParseOptions!;
+                    var parseOptions = (CSharpParseOptions)solution.GetProject(projectId)?.ParseOptions!;
```

Comment:
❕ The previous code was correct. If you do change to `GetProject` (which would only be due to the lack of availability of `GetRequiredProject`), this code should use `!.` instead of `?.`.

```suggestion
                    var parseOptions = (CSharpParseOptions)solution.GetProject(projectId)!.ParseOptions!;
```

Reply:
Let me try to move these extensions methods into shared layer, presuming they are going to be helpful in future. I'll revert these changes.